cjwmodule/http/errors.py

```diff
@@ -4,7 +4,7 @@
 
 
 def TODO_i18n(text: str) -> I18nMessage:
-    return I18nMessage("TODO_i18n", {"text": text})
+    return I18nMessage("TODO_i18n", {"text": text}, "cjwmodule")
```

Comment:
Please nix this. There's no clear use case for having a second `TODO_i18n` message, since modules can simply use the application-wide one. if we get a use case, then we can add it ... but there's no need in this pull request.